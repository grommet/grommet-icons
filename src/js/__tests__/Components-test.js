import * as Components from '..';

test('Components loads', () => {
  expect(Components).toMatchSnapshot();
});
