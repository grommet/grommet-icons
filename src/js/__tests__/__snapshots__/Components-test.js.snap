// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components loads 1`] = `
Object {
  "Accessibility": [Function],
  "Achievement": [Function],
  "Action": [Function],
  "Actions": [Function],
  "Ad": [Function],
  "Add": [Function],
  "AddCircle": [Function],
  "Aed": [Function],
  "Aggregate": [Function],
  "Aid": [Function],
  "AidOption": [Function],
  "Alarm": [Function],
  "Alert": [Function],
  "Amazon": [Function],
  "Amex": [Function],
  "Analytics": [Function],
  "Android": [Function],
  "Announce": [Function],
  "Apple": [Function],
  "Apps": [Function],
  "Archive": [Function],
  "Archlinux": [Function],
  "Article": [Function],
  "Aruba": [Function],
  "Ascend": [Function],
  "AssistListening": [Function],
  "Atm": [Function],
  "Attachment": [Function],
  "Attraction": [Function],
  "Baby": [Function],
  "BackTen": [Function],
  "Bar": [Function],
  "BarChart": [Function],
  "Basket": [Function],
  "Bike": [Function],
  "Bitcoin": [Function],
  "BlockQuote": [Function],
  "Blog": [Function],
  "Bold": [Function],
  "Book": [Function],
  "Bookmark": [Function],
  "BottomCorner": [Function],
  "Braille": [Function],
  "Briefcase": [Function],
  "Brush": [Function],
  "Bug": [Function],
  "Bundle": [Function],
  "Bus": [Function],
  "BusinessService": [Function],
  "Cafeteria": [Function],
  "Calculator": [Function],
  "Calendar": [Function],
  "Camera": [Function],
  "Capacity": [Function],
  "Car": [Function],
  "CaretDown": [Function],
  "CaretNext": [Function],
  "CaretPrevious": [Function],
  "CaretUp": [Function],
  "Cart": [Function],
  "Catalog": [Function],
  "CatalogOption": [Function],
  "Centos": [Function],
  "Channel": [Function],
  "ChapterAdd": [Function],
  "ChapterNext": [Function],
  "ChapterPrevious": [Function],
  "Chat": [Function],
  "ChatOption": [Function],
  "Checkbox": [Function],
  "CheckboxSelected": [Function],
  "Checkmark": [Function],
  "Chrome": [Function],
  "CircleInformation": [Function],
  "CirclePlay": [Function],
  "CircleQuestion": [Function],
  "Clear": [Function],
  "ClearOption": [Function],
  "Cli": [Function],
  "Clipboard": [Function],
  "Clock": [Function],
  "Clone": [Function],
  "Close": [Function],
  "ClosedCaption": [Function],
  "Cloud": [Function],
  "CloudComputer": [Function],
  "CloudDownload": [Function],
  "CloudSoftware": [Function],
  "CloudUpload": [Function],
  "Cloudlinux": [Function],
  "Cluster": [Function],
  "CoatCheck": [Function],
  "Code": [Function],
  "Codepen": [Function],
  "Columns": [Function],
  "Compare": [Function],
  "Compass": [Function],
  "Compliance": [Function],
  "Configure": [Function],
  "Connect": [Function],
  "Connectivity": [Function],
  "Contact": [Function],
  "ContactInfo": [Function],
  "Contract": [Function],
  "Copy": [Function],
  "CreativeCommons": [Function],
  "CreditCard": [Function],
  "Css3": [Function],
  "Cube": [Function],
  "Cubes": [Function],
  "Currency": [Function],
  "Cursor": [Function],
  "Cut": [Function],
  "Cycle": [Function],
  "Dashboard": [Function],
  "Database": [Function],
  "Debian": [Function],
  "Deliver": [Function],
  "Deploy": [Function],
  "Descend": [Function],
  "Desktop": [Function],
  "Detach": [Function],
  "Device": [Function],
  "Diamond": [Function],
  "Directions": [Function],
  "Dislike": [Function],
  "Docker": [Function],
  "Document": [Function],
  "DocumentCloud": [Function],
  "DocumentConfig": [Function],
  "DocumentCsv": [Function],
  "DocumentDownload": [Function],
  "DocumentExcel": [Function],
  "DocumentImage": [Function],
  "DocumentLocked": [Function],
  "DocumentMissing": [Function],
  "DocumentNotes": [Function],
  "DocumentOutlook": [Function],
  "DocumentPdf": [Function],
  "DocumentPerformance": [Function],
  "DocumentPpt": [Function],
  "DocumentRtf": [Function],
  "DocumentSound": [Function],
  "DocumentStore": [Function],
  "DocumentTest": [Function],
  "DocumentText": [Function],
  "DocumentThreat": [Function],
  "DocumentTime": [Function],
  "DocumentTransfer": [Function],
  "DocumentTxt": [Function],
  "DocumentUpdate": [Function],
  "DocumentUpload": [Function],
  "DocumentUser": [Function],
  "DocumentVerified": [Function],
  "DocumentVideo": [Function],
  "DocumentWindows": [Function],
  "DocumentWord": [Function],
  "DocumentZip": [Function],
  "Domain": [Function],
  "Dos": [Function],
  "Down": [Function],
  "Download": [Function],
  "Drag": [Function],
  "Drawer": [Function],
  "DriveCage": [Function],
  "Dropbox": [Function],
  "Duplicate": [Function],
  "Dxc": [Function],
  "Edge": [Function],
  "Edit": [Function],
  "Eject": [Function],
  "Elevator": [Function],
  "Emergency": [Function],
  "Emoji": [Function],
  "EmptyCircle": [Function],
  "Erase": [Function],
  "Escalaton": [Function],
  "Expand": [Function],
  "Facebook": [Function],
  "FacebookOption": [Function],
  "Fan": [Function],
  "FastForward": [Function],
  "Favorite": [Function],
  "Fedora": [Function],
  "Filter": [Function],
  "FingerPrint": [Function],
  "Fireball": [Function],
  "Firefox": [Function],
  "Flag": [Function],
  "Folder": [Function],
  "FolderCycle": [Function],
  "FolderOpen": [Function],
  "FormAdd": [Function],
  "FormAttachment": [Function],
  "FormCalendar": [Function],
  "FormCheckmark": [Function],
  "FormClock": [Function],
  "FormClose": [Function],
  "FormCut": [Function],
  "FormDown": [Function],
  "FormEdit": [Function],
  "FormFilter": [Function],
  "FormFolder": [Function],
  "FormLocation": [Function],
  "FormLock": [Function],
  "FormNext": [Function],
  "FormNextLink": [Function],
  "FormPrevious": [Function],
  "FormPreviousLink": [Function],
  "FormRefresh": [Function],
  "FormSchedule": [Function],
  "FormSearch": [Function],
  "FormSubtract": [Function],
  "FormTrash": [Function],
  "FormUp": [Function],
  "FormUpload": [Function],
  "FormView": [Function],
  "FormViewHide": [Function],
  "ForwardTen": [Function],
  "Freebsd": [Function],
  "Gallery": [Function],
  "Gamepad": [Function],
  "Gift": [Function],
  "Github": [Function],
  "Globe": [Function],
  "Golang": [Function],
  "Google": [Function],
  "GooglePlus": [Function],
  "GoogleWallet": [Function],
  "Gremlin": [Function],
  "Grid": [Function],
  "Grommet": [Function],
  "Group": [Function],
  "Grow": [Function],
  "Hadoop": [Function],
  "Halt": [Function],
  "Help": [Function],
  "Heroku": [Function],
  "History": [Function],
  "Home": [Function],
  "Horton": [Function],
  "Host": [Function],
  "HostMaintenance": [Function],
  "Hp": [Function],
  "Hpe": [Function],
  "HpeLabs": [Function],
  "Hpi": [Function],
  "Html5": [Function],
  "IceCream": [Function],
  "Image": [Function],
  "Impact": [Function],
  "InProgress": [Function],
  "Inbox": [Function],
  "Indicator": [Function],
  "Info": [Function],
  "Inherit": [Function],
  "Inspect": [Function],
  "Instagram": [Function],
  "Install": [Function],
  "InstallOption": [Function],
  "Integration": [Function],
  "InternetExplorer": [Function],
  "Italic": [Function],
  "Iteration": [Function],
  "Java": [Function],
  "Language": [Function],
  "Launch": [Function],
  "Layer": [Function],
  "License": [Function],
  "Like": [Function],
  "LineChart": [Function],
  "Link": [Function],
  "LinkBottom": [Function],
  "LinkDown": [Function],
  "LinkNext": [Function],
  "LinkPrevious": [Function],
  "LinkTop": [Function],
  "LinkUp": [Function],
  "Linkedin": [Function],
  "LinkedinOption": [Function],
  "Local": [Function],
  "Location": [Function],
  "LocationPin": [Function],
  "Lock": [Function],
  "Login": [Function],
  "Logout": [Function],
  "Lounge": [Function],
  "Magic": [Function],
  "Mail": [Function],
  "MailOption": [Function],
  "Mandriva": [Function],
  "Manual": [Function],
  "Map": [Function],
  "MapLocation": [Function],
  "Mastercard": [Function],
  "Medium": [Function],
  "Menu": [Function],
  "Microfocus": [Function],
  "Microphone": [Function],
  "Money": [Function],
  "Monitor": [Function],
  "Monospace": [Function],
  "More": [Function],
  "Multimedia": [Function],
  "Multiple": [Function],
  "Music": [Function],
  "Mysql": [Function],
  "Navigate": [Function],
  "Network": [Function],
  "New": [Function],
  "NewWindow": [Function],
  "Next": [Function],
  "Nodes": [Function],
  "Norton": [Function],
  "Note": [Function],
  "Notes": [Function],
  "Notification": [Function],
  "ObjectGroup": [Function],
  "ObjectUngroup": [Function],
  "OfflineStorage": [Function],
  "Onedrive": [Function],
  "Opera": [Function],
  "Optimize": [Function],
  "Oracle": [Function],
  "OrderedList": [Function],
  "Organization": [Function],
  "Overview": [Function],
  "Paint": [Function],
  "Pan": [Function],
  "Pause": [Function],
  "PauseFill": [Function],
  "Paypal": [Function],
  "Performance": [Function],
  "PersonalComputer": [Function],
  "PieChart": [Function],
  "PiedPiper": [Function],
  "Pin": [Function],
  "Pinterest": [Function],
  "Plan": [Function],
  "Play": [Function],
  "PlayFill": [Function],
  "Power": [Function],
  "Previous": [Function],
  "Print": [Function],
  "ProductHunt": [Function],
  "Radial": [Function],
  "RadialSelected": [Function],
  "Raspberry": [Function],
  "Reactjs": [Function],
  "Reddit": [Function],
  "Redhat": [Function],
  "Redo": [Function],
  "Refresh": [Function],
  "Resources": [Function],
  "Restaurant": [Function],
  "Restroom": [Function],
  "RestroomMen": [Function],
  "RestroomWomen": [Function],
  "Resume": [Function],
  "Revert": [Function],
  "Rewind": [Function],
  "Risk": [Function],
  "Robot": [Function],
  "RotateLeft": [Function],
  "RotateRight": [Function],
  "Rss": [Function],
  "Run": [Function],
  "SafariOption": [Function],
  "Satellite": [Function],
  "Save": [Function],
  "Scan": [Function],
  "Schedule": [Function],
  "ScheduleNew": [Function],
  "SchedulePlay": [Function],
  "Schedules": [Function],
  "Sco": [Function],
  "Scorecard": [Function],
  "Search": [Function],
  "SearchAdvanced": [Function],
  "Secure": [Function],
  "Select": [Function],
  "Selection": [Function],
  "Semantics": [Function],
  "Send": [Function],
  "Server": [Function],
  "ServerCluster": [Function],
  "Servers": [Function],
  "ServicePlay": [Function],
  "Services": [Function],
  "SettingsOption": [Function],
  "Share": [Function],
  "ShareOption": [Function],
  "Shield": [Function],
  "ShieldSecurity": [Function],
  "Shift": [Function],
  "Shop": [Function],
  "Sidebar": [Function],
  "Sign": [Function],
  "Skype": [Function],
  "Slack": [Function],
  "Snapchat": [Function],
  "Solaris": [Function],
  "Sort": [Function],
  "Spa": [Function],
  "Split": [Function],
  "Splits": [Function],
  "Square": [Function],
  "StackOverflow": [Function],
  "Stakeholder": [Function],
  "Star": [Function],
  "StarHalf": [Function],
  "StatusCritical": [Function],
  "StatusCriticalSmall": [Function],
  "StatusDisabled": [Function],
  "StatusDisabledSmall": [Function],
  "StatusGood": [Function],
  "StatusGoodSmall": [Function],
  "StatusInfo": [Function],
  "StatusInfoSmall": [Function],
  "StatusPlaceholder": [Function],
  "StatusPlaceholderSmall": [Function],
  "StatusUnknown": [Function],
  "StatusUnknownSmall": [Function],
  "StatusWarning": [Function],
  "StatusWarningSmall": [Function],
  "Steps": [Function],
  "StepsOption": [Function],
  "Stop": [Function],
  "StopFill": [Function],
  "Storage": [Function],
  "StreetView": [Function],
  "StrikeThrough": [Function],
  "Stripe": [Function],
  "Subscript": [Function],
  "Subtract": [Function],
  "SubtractCircle": [Function],
  "Superscript": [Function],
  "Support": [Function],
  "Suse": [Function],
  "Swift": [Function],
  "Swim": [Function],
  "Sync": [Function],
  "System": [Function],
  "Table": [Function],
  "TableAdd": [Function],
  "Tag": [Function],
  "Tape": [Function],
  "TapeOption": [Function],
  "Target": [Function],
  "Task": [Function],
  "Tasks": [Function],
  "Technology": [Function],
  "Template": [Function],
  "Terminal": [Function],
  "Test": [Function],
  "TestDesktop": [Function],
  "TextAlignCenter": [Function],
  "TextAlignFull": [Function],
  "TextAlignLeft": [Function],
  "TextAlignRight": [Function],
  "TextWrap": [Function],
  "Threats": [Function],
  "ThreeD": [Function],
  "ThreeDffects": [Function],
  "Ticket": [Function],
  "Tip": [Function],
  "Toast": [Function],
  "Tools": [Function],
  "Tooltip": [Function],
  "TopCorner": [Function],
  "Train": [Function],
  "Transaction": [Function],
  "Trash": [Function],
  "Tree": [Function],
  "TreeOption": [Function],
  "Trigger": [Function],
  "Trophy": [Function],
  "Troubleshoot": [Function],
  "Tty": [Function],
  "Tumblr": [Function],
  "Turbolinux": [Function],
  "Twitter": [Function],
  "Ubuntu": [Function],
  "Underline": [Function],
  "Undo": [Function],
  "Unlink": [Function],
  "Unlock": [Function],
  "UnorderedList": [Function],
  "Up": [Function],
  "Update": [Function],
  "Upgrade": [Function],
  "Upload": [Function],
  "User": [Function],
  "UserAdd": [Function],
  "UserAdmin": [Function],
  "UserExpert": [Function],
  "UserFemale": [Function],
  "UserManager": [Function],
  "UserNew": [Function],
  "UserPolice": [Function],
  "UserSettings": [Function],
  "UserWorker": [Function],
  "Validate": [Function],
  "Vend": [Function],
  "Video": [Function],
  "View": [Function],
  "Vimeo": [Function],
  "VirtualMachine": [Function],
  "VirtualStorage": [Function],
  "Visa": [Function],
  "VmMaintenance": [Function],
  "Vmware": [Function],
  "Volume": [Function],
  "VolumeControl": [Function],
  "VolumeLow": [Function],
  "VolumeMute": [Function],
  "Vulnerability": [Function],
  "Waypoint": [Function],
  "Wheelchair": [Function],
  "WheelchairActive": [Function],
  "Windows": [Function],
  "WindowsLegacy": [Function],
  "Wordpress": [Function],
  "Workshop": [Function],
  "Yoga": [Function],
  "Youtube": [Function],
  "ZoomIn": [Function],
  "ZoomOut": [Function],
  "__esModule": true,
  "default": [Function],
}
`;
